workspaces/acr/plugins/acr/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@backstage-community/plugin-acr",
+  "version": "1.7.15",
```

Comment:
Should this be a larger number than the values in janus-idp namespace? https://www.npmjs.com/package/@janus-idp/backstage-plugin-acr?activeTab=versions is at 1.8.0 already after I bumped everything in janus plugins main branch. How about 1.9? 
```suggestion
  "version": "1.9.0",
```